openfisca_us/variables/usda/wic/is_wic_eligible.py

```diff
@@ -0,0 +1,24 @@
+from openfisca_us.model_api import *
+
+
+class is_wic_eligible(Variable):
```

Comment:
Per our discussion we might want to just make this `wic` and assign the estimated values here. If we did it separately, we'd effectively be duplicating some `wic_category` logic.